rpcs3/rpcs3qt/emu_settings_type.h

```diff
@@ -11,7 +11,7 @@ enum class emu_settings_type
 	PPUDecoder,
 	SPUDecoder,
 	HookStaticFuncs,
-	EnableThreadScheduler,
+	ThreadSchedulerMode,
```

Comment:
This is missing handling in emu_settings::GetLocalizedSetting, since it is now ComboBox with translatable options.

Reply:
Done.